osu.Game/osu.Game.csproj

```diff
@@ -19,6 +19,7 @@
   </ItemGroup>
   <ItemGroup Label="Package References">
     <PackageReference Include="DiffPlex" Version="1.7.0" />
+    <PackageReference Include="HtmlAgilityPack" Version="1.11.33" />
```

Comment:
@smoogipoo any opinions on whether we should be adding an html parser for this purpose?

The library is only 1.3mb, so it's not a *huge* overhead, but if we so chose to we could use simple regex tokenisation for this specific purpose (the content being parsed is in a pretty simple format for now: https://raw.githubusercontent.com/ppy/osu-wiki/master/wiki/Main_Page/en.md).

I can imagine that we will need more html parsing in the future though, so maybe the library usage is fine?

Reply:
Sounds fine.